erpnext/stock/reorder_item.py

```diff
@@ -149,11 +150,16 @@ def _log_exception():
 							conversion_factor = frappe.db.get_value("UOM Conversion Detail",
 								{'parent': item.name, 'uom': uom}, 'conversion_factor') or 1.0
 
+					must_be_whole_number = frappe.get_value("UOM", uom, "must_be_whole_number")
```

Comment:
```suggestion
					must_be_whole_number = frappe.db.get_value("UOM", uom, "must_be_whole_number", cache=True)
```

This will probably get called 100s of time in same DB transaction. Better to cache the results.